app/controllers/checksum_records_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ChecksumRecordsController < ApplicationController
+  before_action :set_checksum_record, only: [:show, :update, :destroy]
+
+  # GET /checksum_records
+  # GET /checksum_records.json
+  def index
+    @checksum_records = ChecksumRecord.all
+  end
+
+  # GET /checksum_records/1
+  # GET /checksum_records/1.json
+  def show
+  end
+
+  # POST /checksum_records
+  # POST /checksum_records.json
+  def create
+    @checksum_record = ChecksumRecord.new(checksum_record_params)
+
+    if @checksum_record.save
+      render :show, status: :created, location: @checksum_record
+    else
+      render json: @checksum_record.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /checksum_records/1
+  # PATCH/PUT /checksum_records/1.json
+  def update
```

Comment:
I think we don't want "update" per "specs"

Reply:
https://docs.google.com/document/d/18MPHfUtEwNrCp2idmSt-TvDnxF3P0Kt3WWWnN5pBZ7M/edit#heading=h.yzqsqntox27w

Reply:
Tommy created issue #11 to deal with this.